runtime/autoload/man.vim

```diff
@@ -125,8 +125,8 @@ function! s:put_page(page) abort
   setlocal noreadonly
   silent keepjumps %delete _
   silent put =a:page
-  " Remove all backspaced characters.
-  execute 'silent keeppatterns keepjumps %substitute,.\b,,e'.(&gdefault?'':'g')
+  " Remove all backspaced/escape characters.
+  execute 'silent keeppatterns keepjumps %substitute,.\b\|\e\[\d\+m,,e'.(&gdefault?'':'g')
```

Comment:
I tested it again and it works fine. Also, in `:h /bar` it says `A pattern is one or more branches, separated by "\|".` so it shouldn't match `\b[\d\+m`.